eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Tally.hs

```diff
@@ -184,7 +184,7 @@ tallyTransition = do
                   st'
         updateState st'' xs
 
-  updateState st $ Seq.zip govProcedures [0 ..]
+  updateState st $ Seq.zip govProcedures (fmap GovernanceActionIx [0 .. fromIntegral (Seq.length govProcedures - 1)])
```

Comment:
`GovernanceActionIx` has a `Num` instance, so no extra conversions is needed.
```suggestion
  updateState st $ Seq.zip govProcedures [0 .. fromIntegral (Seq.length govProcedures - 1)]
```

Reply:
Would it be maybe better to do 
`Seq.zip govProcedures (Seq.fromList [0 .. fromIntegral (Seq.length govProcedures - 1)])` 
and remove the OverloadedLists ?

Reply:
They are one and the same thing. This is exactly what `OverloadedLists` will translate it into. So it is up to you.